### DISCUSSION TITLE
Tidak bisa view message api wa

### DISCUSSION THREAD

**Discussion by galangex** [kirimwa-id/forum#98] [Tanya Jawab]
apakah ada yang keliru kok tidak bisa menampilkan status pengiriman pesan, mohon bantuanya 

skrip api

<img width="620" alt="Jepretan Layar 2022-02-26 pukul 19 13 59" src="https://user-images.githubusercontent.com/100219787/155842866-0af9398e-1c62-4302-9fc3-5034cd567a23.png">

respon yang ditampilkan

<img width="492" alt="Jepretan Layar 2022-02-26 pukul 19 14 31" src="https://user-images.githubusercontent.com/100219787/155842871-de346189-8018-44d7-8989-2703307da5ee.png">

**Comment by rioastamal** [comment]
Sebuah environment variable tidak boleh ada whitespace sebelum dan sesudah tanda `=`.

```sh
MESSAGE_ID=YOUR_MESSAGE_ID
```

**Reply by galangex** [reply]
makasih bang, worked
